docs/quickstart.md

```diff
@@ -0,0 +1,270 @@
+## Longleaf basic usage tutorial
+
+This tutorial provides an introduction to the basic functionality of Longleaf by demonstrating how to configure and execute a small set of preservation tasks, using an example data directory as a local sandbox on your own computer.
+
+**Longleaf basic usage tasks covered in this tutorial:**
+
+* Validate the mandatory Longleaf configuration file that contains the storage locations and preservation activities you will use
+* Register example data files
+* Validate metadata for registered files
+* Run a Preservation action on example data files
+* Alter an example data file to cause an error; catch the error by re-running the Preservation action
+* Re-register the altered file, and preserve the file in its new, altered state
+
+### System Requirements for this tutorial
+Longleaf scripts rely on common UNIX programs. In Mac OS X and Linux operating systems, these programs will likely already be installed, but some of these tools, such as `rsync` may be missing from a Windows system unless you have installed it.
+
+### Example data directory for this tutorial
+The example data directory for this tutorial is named 'll-example'; it is located (at the top level within this repository).
```

Comment:
This either needs to be changed or ll-example needs to be moved to the top level. Thoughts?

Reply:
I'm fine with either. Do you have a preference for where 'll-example/' should go?

Reply:
After discussing, we decided to leave ll-example in docs/. I updated the wording in quickstart.md so it correctly describes the location of the example data directory.